rllib/utils/actor_manager.py

```diff
@@ -696,10 +703,30 @@ def fetch_ready_async_reqs(
             The values may be actual data returned or exceptions raised during the
             remote call in the format of RemoteCallResults.
         """
+        if tags == "default":
+            pass
+        elif isinstance(tags, str):
+            tags = {tags}
+        elif isinstance(tags, list):
+            tags = set(tags)
+        else:
+            raise ValueError(
+                f"tags must be either a str or a list of str, got {type(tags)}."
+            )
```

Comment:
hey nit picking comment, can you extract this into a private helper, so it doesn't bloat the fetch_ready_async_reqs() logic? also, how about we use a default value of ``()`` instead? 
for example:
```
def __filter_calls_by_tag(self, tags):
  if isinstance(tags, str):
    tags = {tags}
  else:
    tags= set(tags)

  return {
    call: id for call, (tag, id) in self.__in_flight_req_to_actor_id.items()
    if not tags or tag in tags
  }

def fetch_ready_async_reqs(
    self,
    *,
    tags: Union[str, List[str]] = (),
):
    filtered_calls = self.__filter_calls_by_tag(tags)
    ready, remote_results = self.__fetch_result(
      remote_actor_ids=list(filtered_calls.values()),
	  remote_calls=list(filtered_calls.keys()),
    )
    ...
```

Reply:
sure

Reply:
does it matter to you if `__filter_calls_by_tag` returns a mapping? I can do it that way but out of convenience I've just returned 3 lists instead where the lists are the calls, actor ids, and tags and their indexes correspond to each other.

Reply:
addressed.